recipe/meta.yaml

```
@@ -125,33 +54,162 @@ build:
 
 # Required for conda to generate the correct build matrix
 requirements:
-  build: {{- build_dependencies()|replace("\n  ", "\n    ") }}
-  host: {{- host_dependencies()|replace("\n  ", "\n    ") }}
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake >=3.9.0
+    - git
+    - pkg-config
+    - sed
+    - make
+    # Required for OpenGL support
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - libblas
+    - libcblas
+    - {{ compiler('c') }}  # [linux]
+    - {{ compiler('cxx') }}  # [linux]
+    - {{ compiler('fortran') }}  # [linux]
+    - numpy >=1.11
+    - afterimage
+    - cfitsio >=3.280
+    - clangdev ={{ clang_version }}
+    - clang_variant * cling_{{ clang_patches_version }}
+    - davix >=0.7.2
+    - fftw >=3.3.8
+    - freetype >=2.6.1
+    - gdk-pixbuf >=2.36.9
+    - giflib >=5.1.2
+    - glew >=2.0.0
+    - graphviz >=2.38.0
+    - gsl >=1.10
+    - krb5 >=1.16  # [osx]
+    - libglu  # [linux]
+    - libpng >=1.6
+    - librsvg >=2.44
+    - libtiff >=4.0
+    - libxml2 >=2.9
+    - llvmdev ==5
+    - lz4-c >=1.8
+    # - mysql
+    - openssl
+    - pcre >=8.39
+    # - postgresql
+    - pythia8 >=8.240
+    - python
+    - qt
+    - sqlite >=3.20
+    - tbb >=2019.8
+    - tbb-devel >=2019.8
+    - vdt >=0.4.3
+    - xorg-libx11  # [linux]
+    - xorg-libxau  # [linux]
+    - xorg-libxcursor  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libxfixes  # [linux]
+    - xorg-libxft  # [linux]
+    - xorg-libxpm  # [linux]
+    - xrootd >=4.10.0
+    - xz >=5.2
+    # FIXME: The generated allDict.cxx.pch is dependent on version of the C++ headers used
+    - {{ pin_compatible('libcxx', min_pin='x', max_pin='x') }}
 
 outputs:
   - name: root-dependencies
     build:
       number: {{ build_number }}
+      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ build_number }}
       skip: True  # [win]
       script: echo "Minimal build script is required for conda to set RECIPE_DIR"
       run_exports:
         # https://abi-laboratory.pro/?view=timeline&l=root
         - {{ pin_subpackage('root-dependencies', max_pin='x.x.x') }}
     requirements:
```

```suggestion
     requirements:
     <<: *requirements
```

And then use it to start each of these (overriding either of the two keys as needed with new lists)

---

That also triggers the bug unfortunately. I've reported it in `conda-build`: https://github.com/conda/conda-build/issues/3749